services/docker-compose.yml

```diff
@@ -308,7 +308,7 @@ services:
       resources:
         limits:
           cpus: "1.0"
-          memory: 200M
+          memory: 1024M
```

Comment:
hmm question, how is this handled on machines that might be running 6 s4l-lite?

Reply:
This is the peak memory. It is not a reservation which means it requires at least. 1024MB. Reservation is not specified to avoid the issue.

Reply:
well it still means it might use them.